core/experiments/manager.php

```diff
@@ -444,13 +444,6 @@ private function add_default_features() {
 			'default' => self::STATE_INACTIVE,
 		] );
 
-		$this->add_feature( [
```

Comment:
I am still checking with @Danitsh and @doronwolf if we remove the default loading of the Swiper styles for all websites or for new websites only?

Reply:
In my opinion we need to remove it to "all" the websites, as it was the initial task. Otherwise, why did we do that?